src/lib/parser.js

```diff
@@ -128,9 +237,12 @@ export default function parse(table, config) {
     const resultsMap = toResultsWithRegExp(settings.results);
     const resultsMapCount = resultsMap.length;
     const columnsWithResultsFilter = getFilterForColumnsWithResults(rows, settings, resultsMap);
-    const results = {};
+    const columnsForNameFilter = settings.displayOpponentNameHint ? getFilterForColumnsWithName(rows, settings) : (cell, index) => false;
+    const results = [];
 
-    function parseGames(player, cells) {
+    function parseGames(player, cells, players, settings) {
+        const displayOpponentNameHint = settings.displayOpponentNameHint;
+        const classes = toPrefixedClasses(settings);
```

Comment:
I don't think that we need to run toPrefixedClasses() function for every game cell.
On the contrary - I would move this line to 242 - so it's done once, not for every row.

What do toy think?